ssm.tf

```diff
@@ -0,0 +1,316 @@
+resource "aws_ssm_parameter" "dynamo_table_name_prefix" {
+  name  = "/${var.parameter_prefix}-${local.application_id}/Config/DynamoTableNamePrefix"
```

Comment:
Consider making the prefix (`/${var.parameter_prefix}-${local.application_id}/Config`) a local.